tests/managed-service/test_ms_upgrade.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from ocs_ci.ocs.resources.pod import cal_md5sum
+from ocs_ci.helpers.managed_services import verify_provider_topology
+from ocs_ci.framework.testlib import (
+    pre_upgrade,
+    post_upgrade,
+    ms_consumer_required,
+    ms_provider_required,
+)
+
+logger = logging.getLogger(name=__file__)
+
+
+@pre_upgrade
+@ms_consumer_required
+def test_prepare_block_md5_before_upgrade(block_md5):
+    """
+    Prepare md5 results for utilized RBD PVC.
+
```

Comment:
Maybe I would add here a log message: `logger.info(f"RBD file md5 before upgrade: {block_md5}")`

Reply:
md5 strings are logged when they are taken in respective fixtures

Reply:
Yes, it's not mandatory.